dendrite/pipeline.yml

```diff
@@ -35,6 +35,24 @@ steps:
       - docker#v3.0.1:
           image: "golang:1.13"
           mount-buildkite-agent: false
+
+  - command:
+      - "curl -fsSL https://get.docker.com -o get-docker.sh"
+      - "sh get-docker.sh"
+      - "docker build -t complement-dendrite -f build/scripts/Complement.Dockerfile ."
+      - "wget https://github.com/matrix-org/complement/archive/master.tar.gz && tar -xzf master.tar.gz"
+      - "cd complement-master && COMPLEMENT_BASE_IMAGE=complement-dendrite:latest go test -v ./tests"
+    label: "\U0001F9EA Complement"
+    agents:
+      queue: "medium"
+    plugins:
+      - docker#v3.5.0:
+          image: "golang:1.13"
+          mount-buildkite-agent: false
+          propagate-environment: true
```

Comment:
why do we need this? (other than for cargo-culting from other steps which also don't need it?)

Reply:
Because Complement relies on `CI=true` to determine if it should be using `localhost` or the docker `172` IP when communicating with homeservers.

Reply:
ok, a comment explaining that would be good.

Reply:
To be fair I can just manually do this, so will remove this line.